src/main/json/r6OperatorCounters.json

```diff
@@ -1485,6 +1485,15 @@
                     "properties": {
                         "Description": "The Thermal Feedback System on Glaz's OTs-03 can see through Smoke's Gas Grenades."
                     }
+                },
+                {
+                    "id": "141",
+                    "type": "Minor Counter",
+                    "startNode": "12",
+                    "endNode": "9",
+                    "properties": {
+                        "Description": "Dokkaebi can ring Dokkaebi's phone if she was sneaking around nearby."
+                    }

```

Comment:
I think this should be changed this to something like "Dokkaebi can use her Logic Bomb to ring Caveria's phone. This will reveal Caveria's position when roaming even if Silent Step is active."

Reply:
Yep, just noticed I wrote Dokkaebi can ring Dokkaebi's phone...